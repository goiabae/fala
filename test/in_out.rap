  mov %r0, 1023
  read %t0
  printv %t0
  read %t1
  printv %t1
  printf "foobar"
  printv 1337
  printv 0
  printv 1
