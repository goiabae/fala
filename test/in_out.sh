# echo "Hello, world!"
echo 69420
echo 1337
