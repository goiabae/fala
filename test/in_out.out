694201337foobar133701